classes/controller/AdminController.php

```diff
@@ -337,7 +337,7 @@ class AdminControllerCore extends Controller
     protected $row_hover = true;
 
     /** @var string Action to perform : 'edit', 'view', 'add', ... */
-    protected $action;
+    protected $action = '';
```

Comment:
This looks like a BC break for me. The best fix for me is to convert null values before calling strcmp function

Reply:
I can't really see how that is a BC break.

Reply:
Changing a default value **is** a BC break. If you want to force a null value to be a string, you can add `(string) $this->action` before